VndbSharp/Models/Staff/Staff.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Text;
+using Newtonsoft.Json;
+using VndbSharp.Attributes;
+using VndbSharp.Models.Common;
+
+namespace VndbSharp.Models.Staff
+{
+    public class Staff
+    {
+		public UInt32 Id { get; private set; }
+        public String Name { get; private set; }
+        [JsonProperty("original")]
+        public String OriginalName { get; private set; }
+        public Gender? Gender { get; private set; }
+		public String Language { get; private set; }
+        [JsonProperty("links")]
+		public StaffLinks StaffLinks { get; private set; }
+        public String Description { get; private set; }
+		//TODO: Need to fix staff aliases from breaking
+        public ReadOnlyCollection<StaffAliases> Aliases { get; private set; }
```

Comment:
Why exactly is this broken? I can't exactly accept this PR when something is broken. Whats the exception?